AUTHORS

```diff
@@ -14,7 +14,7 @@ Current Contributors:
 * Erik Bjorge (@ErikBjorge)
 * Ignacio Hernandez (@iherna2)
 * Brent Holtsclaw (@BrentHoltsclaw)
-* Moonjung Kyung (@Mjkyung)
+* Gabriel Kerneis (@kerneis-anssi) (AMD)
```

Comment:
Please change (AMD) to (AMD architecture) or (AMD support) to avoid the misconception that I'd be employed by AMD.

Reply:
I've made the change to AMD support. Thanks @kerneis-anssi !